create-appimage.sh

```diff
@@ -14,37 +17,51 @@ else
     qmake CONFIG+=appimage CONFIG+=qmlui
 fi
 
-make -j8
+NUM_CPUS=`nproc` || true
+if [ -z "$NUM_CPUS" ]; then
+    NUM_CPUS=8
+fi
+
+make -j$NUM_CPUS
+make check
 
-mkdir $TARGET_DIR
+if [ ! -d "$TARGET_DIR" ]; then
+    mkdir $TARGET_DIR
+fi
 make INSTALL_ROOT=$TARGET_DIR install
 
 strip $TARGET_DIR/usr/bin/qlcplus-qml
-strip $TARGET_DIR/usr/lib/libqlcplusengine.so.1.0.0
+# see variables.pri, where to find the LIBSDIR
+find $TARGET_DIR/usr/lib/ -name libqlcplusengine.so.1.0.0 -exec strip -v {} \;
 
-chrpath -r "../lib" $TARGET_DIR/usr/bin/qlcplus-qml
+# FIXME: no rpath or runpath tag found.
+chrpath -r "../lib" $TARGET_DIR/usr/bin/qlcplus-qml || true
 
-cd $TARGET_DIR/usr/bin
+pushd $TARGET_DIR/usr/bin
 find . -name plugins.qmltypes -type f -delete
 find . -name *.qmlc -type f -delete
 rm -rf QtQuick/Extras QtQuick/Particles.2 QtQuick/XmlListModel
 rm -rf QtQuick/Controls.2/designer QtQuick/Controls.2/Material
 rm -rf QtQuick/Controls.2/Universal QtQuick/Controls.2/Fusion
 rm -rf QtQuick/Controls.2/Imagine QtQuick/Controls.2/Scene2D
-cd -
+popd
 
-wget -c https://github.com/AppImage/AppImageKit/releases/download/continuous/AppRun-x86_64 -O $TARGET_DIR/AppRun
-chmod a+x $TARGET_DIR/AppRun
+if [ ! -f $TARGET_DIR/AppRun]; then
```

Comment:
I would remove this check.
There might be a new version of the tool available.

Reply:
OK, check is now removed.